client/components/PubPreview/ManyAuthorsByline.tsx

```diff
@@ -18,27 +17,26 @@ const ManyAuthorsByline = (props: Props) => {
 	const authors = getAllPubContributors(pubData, 'contributors', false, true);
 	const isTruncating = authors.length > truncateAt;
 
+	// If arcadia, use A-Z byline option
+	const bylineOption =
+		pubData.communityId === '1a71ef4d-f6fe-40d3-8379-42fa2141db58'
+			? BylineOption.Az
+			: BylineOption.By;
+
 	if (authors.length === 0) {
 		return null;
 	}
 
 	if (isExpanded && isTruncating) {
 		return (
 			<>
-				<Byline
-					contributors={authors}
-					bylinePrefix={`by ${authors.length} ${naivePluralize(
```

Comment:
What are the consequences of removing this custom `bylinePrefix` logic and the `renderUserLabel` prop?